examples/example.php

```diff
@@ -2,46 +2,80 @@
 
 require "vendor/autoload.php";
 
+use Momento\Auth\EnvMomentoTokenProvider;
 use Momento\Cache\SimpleCacheClient;
 
 $MOMENTO_AUTH_TOKEN = getenv("MOMENTO_AUTH_TOKEN");
-$CACHE_NAME = "cache";
+$CACHE_NAME = getenv("CACHE_NAME");
 $ITEM_DEFAULT_TTL_SECONDS = 60;
 $KEY = "MyKey";
 $VALUE = "MyValue";
 
-function printBanner(string $message) : void {
+function printBanner(string $message): void
+{
     $line = "******************************************************************";
     print "$line\n$message\n$line\n";
 }
 
-function createCache(SimpleCacheClient $client, string $cacheName) : void {
-    try {
-        $client->createCache($cacheName);
-    } catch (\Momento\Cache\Errors\AlreadyExistsError $e) {}
+printBanner("*                      Momento Example Start                     *");
+// Setup
+$authProvider = new EnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN");
+$client = new SimpleCacheClient($authProvider, $ITEM_DEFAULT_TTL_SECONDS);
+
+// Ensure test cache exists
+$response = $client->createCache($CACHE_NAME);
+if ($response->asSuccess()) {
+    print "Created cache " . $CACHE_NAME . "\n";
+}
+if ($response->asAlreadyExists()) {
+    print "Cache " . $CACHE_NAME . " already exists.\n";
+}
+if ($response->asError()) {
+    print "Error creating cache: " . $response->asError()->message() . "\n";
+    exit;
```

Comment:
Same as above, I think this would be clearer as a single if/elseif rather than separate ifs.